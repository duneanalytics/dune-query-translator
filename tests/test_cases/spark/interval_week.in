select now() - interval '64' week