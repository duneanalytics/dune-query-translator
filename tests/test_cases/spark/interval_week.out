/* Success! If you're still running into issues, check out https://dune.com/docs/query/syntax-differences/ or reach out in the #dune-sql Discord channel. */

SELECT
  CURRENT_TIMESTAMP - INTERVAL '448' day /* week doesn't work in DuneSQL */